lib/galaxy/config/__init__.py

```diff
@@ -589,6 +589,17 @@ def __init__(self, **kwargs):
         self.object_store_cache_path = resolve_path(kwargs.get("object_store_cache_path", "database/object_store_cache"), self.data_dir)
         self.object_store_store_by = kwargs.get("object_store_store_by", "id")
 
+        # Configuration options for new storage media uses can plug-in.
+        self.default_storage_media_jobs_directory = kwargs.get(
+            "default_storage_media_jobs_directory",
+            "database/job_working_directory_storage_media")
```

Comment:
This is not how we use the database directory any more, I get that object_store_cache_path up there is also broken so I opened https://github.com/galaxyproject/galaxy/pull/8490 to try to fix it.

Reply:
Good point, updated.